third_party/move/move-binary-format/src/file_format_common.rs

```diff
@@ -518,9 +518,9 @@ pub const VERSION_8: u32 = 8;
 pub const VERSION_MAX: u32 = VERSION_7;
 
 /// Mark which version is the default version. This is the version used by default by tools like
-/// the compiler. Notice that this version might be different than the one supported on nodes.
+/// the compiler. Notice that this version might be different from the one supported on nodes.
```

Comment:
FYI: "different than" is perfectly acceptable in American English, although it seems that "different from" is preferred in writing.  Wasn't really worth changing, but not worth arguing about, either.

Brits randomly use "different to" which always sounds very wrong to me.

Reply:
Spell checker made this suggestion.